netlify/functions/render.js

```diff
@@ -20,7 +20,7 @@ async function getLayerTokens(eventId, url) {
 		}));
 	return res;
 }
-
+// TODO: Refactor to ask these information to our API
```

Comment:
If we are going to proxy every call to Goldsky, we should provide an endpoint to fulfil these request and not expose our API keys